vignettes/stability.Rmd

```diff
@@ -87,8 +87,18 @@ To make these ideas concrete, lets apply them to a few base functions:
     vec_ptype(c(Sys.Date(), NA))
     ```
     
-    `c()` is length-stable because `length(c(x, y))` is always 
-    `length(x) + length(y)`.
+    `c()` is *almost always* length-stable because `length(c(x, y))`
+    *almost always* equals `length(x) + length(y)`. One common source of
+    instability here is dealing with non-vectors (see also later section
+    "Non-vectors"):
+    
+    ```{r}
+    env <- new.env()
```

Comment:
Can you please add `parent = emptyenv()` here? Then I think you can skip the comment below.

Reply:
Done.